src/ec/packages/ec-component-breadcrumb-standardised/__snapshots__/breadcrumb-standardised.test.js.snap

```diff
@@ -33,14 +33,13 @@ exports[`EC - Breadcrumb Standardised Long renders correctly 1`] = `
       </svg>
     </li>
     <li
-      aria-hidden="false"
       class="ecl-breadcrumb-standardised__segment ecl-breadcrumb-standardised__segment--ellipsis"
-      data-ecl-breadcrumb-standardised-ellipsis="true"
+      data-ecl-breadcrumb-standardised-ellipsis=""
     >
       <button
         aria-label="Click to expand"
         class="ecl-breadcrumb-standardised__ellipsis"
-        data-ecl-breadcrumb-standardised-ellipsis-button="true"
+        data-ecl-breadcrumb-standardised-ellipsis-button=""
```

Comment:
js controlled

Reply:
same as previous comment.. I was also referring on this issue: https://github.com/ec-europa/ecl-twig/issues/165#issuecomment-539023572
because sometime we set the `true` value, sometime not.